bodhi/server/models.py

```diff
@@ -1756,13 +1764,15 @@ def set_request(self, db, action, username):
                         'stable repository.  It must first reach a karma of %s, consisting of %s '
                         'positive karma from proventesters, along with %d additional karma from '
                         'the community. Or, it must spend %s days in testing without any negative '
-                        'feedback')
+                        'feedback.')
                     stern_note = stern_note % (
                         config.get('critpath.min_karma'),
                         config.get('critpath.num_admin_approvals'),
                         (config.get('critpath.min_karma') -
                             config.get('critpath.num_admin_approvals')),
                         config.get('critpath.stable_after_days_without_negative_karma'))
+                    if config.get('greenwave_enabled'):
+                        stern_note += ' Or, it must pass gating.'
```

Comment:
The "or" makes it sound like passing gating will allow the developer to push the update to stable without karma if it passes gating. With this code, it produces this message:

```
This critical path update has not yet been approved for pushing to the stable repository. It must first reach a karma of 2, consisting of 2 positive karma from proventesters, along with 0 additional karma from the community. Or, it must spend 14 days in testing without any negative feedback. Or, it must pass gating.
```

As I understand it, the automated tests do not supercede the other requirements (karma or time), but are in addition to the other requirements. Thus, I suggest "Additionally, it must pass automated tests." so that it reflects the Fedora policy.